Bottom: 1344f0eabcdb44dc1c4e247d6688445f44e4f1b9
Top:    15ee79b4ec498bd2abc5d8f835a8e9402bf8a1a7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:15:56 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index fc9c7e6..eee83b6 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -471,7 +471,6 @@ object3d_from_board_outline (void)
           face3d_set_normal (faces[i], 1., 0., 0.);  /* A normal to the axis direction */
                                     /* XXX: ^^^ Could line this up with the direction to the vertex in the corresponding circle edge */
 
-
           edge_info_set_round (UNDIR_DATA (edges[i]),
                                COORD_TO_MM (ct->cx), COORD_TO_MM (ct->cy), -COORD_TO_MM (HACK_BOARD_THICKNESS), /* Center of circle */
                                0., 0., 1., /* Normal */ COORD_TO_MM (ct->radius));
